src/main/java/be/acara/frontend/exception/ExceptionController.java

```diff
@@ -1,15 +1,32 @@
 package be.acara.frontend.exception;
 
 import feign.FeignException;
+import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
 
 @ControllerAdvice
+@Controller
 public class ExceptionController {
     
     @ExceptionHandler(FeignException.class)
     public String handleFeignClientExceptions(FeignException fe) {
-        return "redirect:/login";
+        switch (fe.status()) {
+            case 401:
```

Comment:
401 is unauthorized, 403 is forbidden?

Reply:
Yes, both statuses indicate a user isn't allowed to access the resource. This is only for the feignClient calls to the backend. In any other case, the user would get the frontend unauthorized error and would get redirected to /login automatically.